### PR TITLE
Vivo's translation into Russian 

### PR DESCRIPTION
**Pull request by litvinovg**
This PR adds translations in Russian for VIVO.

### CODE REVIEW

File: ru_RU/webapp/src/main/webapp/i18n/vivo_all_ru_RU.properties [hunk 1]
```diff
@@ -0,0 +1,929 @@
...
+data_for = Данные
+refreshing_data_message = сейчас обновляется. Визуализация загрузится, как только мы закончим вычисления, или вы можете искать или просматривать другие данные в VIVO и вернуться через несколько минут.\u0020
+disclaimer_text_one = Эта информация основана только на
+disclaimer_text_two = которые были загружены в систему VIVO к\u0020
+level_undefined_error = УРОВЕНЬ СРАВНЕНИЯ НЕ ЗАДАН
+
+total_number_of = Общее число
+number_of = Число
+have_an_unknown = неизвестная ошибка
+year_not_chartered = год (не указан выше)\u0020
```
Comment (chenejac) [inline ru_RU/webapp/src/main/webapp/i18n/vivo_all_ru_RU.properties:695]: The Unicode \u0020 here with a reason?

**Review by chenejac — CHANGES_REQUESTED** [review]
@litvinovg, similarly as for your VITRO-language pull request, I have tried to "visually" review that, to check whether something is missing, but I am not a Russian speaker, therefore I didn't check translations.

File: ru_RU/webapp/src/main/webapp/i18n/vivo_all_ru_RU.properties (hunk 1 above)
Comment (litvinovg) [reply ru_RU/webapp/src/main/webapp/i18n/vivo_all_ru_RU.properties]: Not this time. Fixed.

File: ru_RU/webapp/src/main/webapp/i18n/vivo_all_ru_RU.properties [hunk 2]
```diff
@@ -0,0 +1,929 @@
...
+in_completed_year = в полном году
+were = было\u0020
```
[the remark this answers is not in the document]
Comment (litvinovg) [reply ru_RU/webapp/src/main/webapp/i18n/vivo_all_ru_RU.properties]: No. Fixed.

File: ru_RU/home/src/main/resources/rdf/i18n/ru_RU/tbox/firsttime/vitroAnnotations_ru_RU.n3 [hunk 3]
```diff
@@ -0,0 +1,565 @@
...
+
+skos:narrower
+      vitro:fullPropertyNameAnnot
+              "более узкое понятие"@ru-RU ;
+      vitro:publicDescriptionAnnot
+              "Это соотносит термин, более узкий по значению (т.е. более конкретный), с другим термином, более широким по значению, когда объем (значение) более узкого термина полностью входит в объем более широкого термина."@ru-RU .
+
+bibo:pageStart
+      vitro:publicDescriptionAnnot
+              "Starting page number within a continuous page range."@ru-RU .
```
[the remark this answers is not in the document]
Comment (litvinovg) [reply ru_RU/home/src/main/resources/rdf/i18n/ru_RU/tbox/firsttime/vitroAnnotations_ru_RU.n3]: Fixed

File: ru_RU/home/src/main/resources/rdf/i18n/ru_RU/tbox/firsttime/vitroAnnotations_ru_RU.n3 [hunk 4]
```diff
@@ -0,0 +1,565 @@
...
+
+vivo:equipmentFor
+      vitro:fullPropertyNameAnnot
+              "оборудование для"@ru-RU ;
+      vitro:publicDescriptionAnnot
+              "Это связывает оборудование с организацией, которой оно принадлежит."@ru-RU .
+
+bibo:prefixName
+      vitro:publicDescriptionAnnot
+              "A title placed before a person's name."@ru-RU .
```
[the remark this answers is not in the document]
Comment (litvinovg) [reply ru_RU/home/src/main/resources/rdf/i18n/ru_RU/tbox/firsttime/vitroAnnotations_ru_RU.n3]: Fixed

File: ru_RU/home/src/main/resources/rdf/i18n/ru_RU/tbox/firsttime/initialTBoxAnnotations_ru_RU.nt [hunk 5]
```diff
@@ -0,0 +1,884 @@
...
+<http://purl.obolibrary.org/obo/ARG_2000077> <http://www.w3.org/2000/01/rdf-schema#label> "Американская коллегия ортопедической хирургии"@ru-RU .
+<http://purl.obolibrary.org/obo/ERO_0000787> <http://www.w3.org/2000/01/rdf-schema#label> "Роль сотрудника"@ru-RU .
+<http://vivoweb.org/ontology/core#publicationVenueFor> <http://www.w3.org/2000/01/rdf-schema#label> "место публикации"@ru-RU .
+<http://purl.org/net/OCRe/study_design.owl#Phase_0> <http://www.w3.org/2000/01/rdf-schema#label> "Фаза 0"@ru-RU .
+<http://vivoweb.org/ontology/scientific-research#Phase2ClinicalTrial> <http://www.w3.org/2000/01/rdf-schema#label> "Фаза 2 клинического испытания"@ru-RU .
+<http://aims.fao.org/aos/geopolitical.owl#codeAGROVOC> <http://www.w3.org/2000/01/rdf-schema#label> "код AGROVOC"@ru-RU .
+<http://purl.org/dc/terms/contributor> <http://www.w3.org/2000/01/rdf-schema#label> "соавтор"@ru-RU .
+<http://purl.obolibrary.org/obo/RO_0002350> <http://www.w3.org/2000/01/rdf-schema#label> "участник"@ru-RU .
+<http://www.w3.org/2006/vcard/ns#telephone> <http://www.w3.org/2000/01/rdf-schema#label> "Телефон"@ru-RU .
```
[the remark this answers is not in the document]
Comment (litvinovg) [reply ru_RU/home/src/main/resources/rdf/i18n/ru_RU/tbox/firsttime/initialTBoxAnnotations_ru_RU.nt:750]: There are both <http://www.w3.org/2006/vcard/ns#telephone> and <http://www.w3.org/2006/vcard/ns#Telephone> in use, so I guess both should be translated.

File: ru_RU/home/src/main/resources/rdf/i18n/ru_RU/tbox/firsttime/vitroAnnotations_ru_RU.n3 [hunk 6]
```diff
@@ -0,0 +1,565 @@
...
+
+vivo:researcherId
+      vitro:publicDescriptionAnnot
+              "The identification number given to the profile created by a researcher in ResearcherID (https://researcherid.com/)."@ru-RU .
```
[the remark this answers is not in the document]
Comment (litvinovg) [reply ru_RU/home/src/main/resources/rdf/i18n/ru_RU/tbox/firsttime/vitroAnnotations_ru_RU.n3]: Fixed

**Review by chenejac — APPROVED** [review]

File: ru_RU/webapp/src/main/webapp/templates/freemarker/lib/lib-home-page_ru_RU.ftl [hunk 7]
```diff
@@ -0,0 +1,199 @@
+<#-- $This file is distributed under the terms of the license in LICENSE$ -->
```
Comment (chenejac) [inline ru_RU/webapp/src/main/webapp/templates/freemarker/lib/lib-home-page_ru_RU.ftl:1]: Is this file needed for localization in Russian?

**Review by tawahle — APPROVED** [review]

**Review by chenejac — APPROVED** [review]

**Pull request merged**